app/apps/views.js

```diff
@@ -162,13 +108,38 @@ exports.app_edit = [
       .all([app_request, buckets_request, users_request])
       .then(function(responses) {
         let [app_response, buckets_response, users_response] = responses;
+        const all_buckets = buckets_response.results;
+        const bucket_ids_associated = app_response.apps3buckets.map(as => as.s3bucket.id);
+        const buckets_other = all_buckets.filter(bucket => {
+          return bucket_ids_associated.indexOf(bucket.id) < 0;
+        });
         let template_args = {
           app: app_response,
-          buckets: buckets_response.results,
+          available_buckets: buckets_other,
```

Comment:
@axemonkey FYI: I'm starting to call these non-associated-stuff variables `*_options` as discussed with @andyhd, e.g.:
- `buckets_options`
- `apps_options`
- `user_options`